ui/public/scripts/app.js

```diff
@@ -43,7 +43,10 @@ var resolveMCPRoute = {
     'ProjectsService',
     'DataService',
     function($route, ProjectsService, DataService) {
-      if (window.MCP_URL) {
+      window.MCP_URL = null;
```

Comment:
@philbrookes Might be worth adding some more info in the comment. e.g. why it gets reset to null, and why it might sometimes lookup a route and other times won't.